localstack-core/localstack/services/sqs/provider.py

```diff
@@ -1478,27 +1481,23 @@ def cancel_message_move_task(
 
     def tag_queue(self, context: RequestContext, queue_url: String, tags: TagMap, **kwargs) -> None:
```

Comment:
Resource tags also need updates when [creating](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_CreateQueue.html) the queue (if [tags](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_CreateQueue.html#SQS-CreateQueue-request-tags) parameter is specified) and when [deleting](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_DeleteQueue.html) the queue